services/ui-src/src/Routes.tsx

```diff
@@ -155,11 +205,11 @@ const ROUTE_LIST: RouteSpec[] = [
         exact: true,
         component: UserPage,
       },
-      {
-        path: ROUTES.PROFILE + "/:userId",
-        exact: true,
-        component: UserPage,
-      },
+      // {
+      //   path: ROUTES.PROFILE + "/:userId",
+      //   exact: true,
+      //   component: UserPage,
+      // },
```

Comment:
We still need this route because admin users should be able to see user profile pages from clicking the link on the User Management page. We just need to inject code to check the token to ensure they have the proper role in the jwt to access this route. Maybe similar to how you locked the User Management page but it would be more complex because we would want to use whatever rules are used to populate the user management page to ensure its only a certain group of users as well. AKA Admin doesnt mean you can see everyone. For example state admins should be be able to see user profiles of users in a state for which they are a state admin.